templates/base/head_opengraph.tmpl

```diff
@@ -0,0 +1,42 @@
+{{if .PageIsUserProfile}}
+	<meta property="og:title" content="{{.ContextUser.DisplayName}}">
+	<meta property="og:type" content="profile">
+	<meta property="og:image" content="{{.ContextUser.AvatarLink ctx}}">
+	<meta property="og:url" content="{{.ContextUser.HTMLURL}}">
+	{{if .ContextUser.Description}}
+		<meta property="og:description" content="{{.ContextUser.Description}}">
+	{{end}}
+{{else if .Repository}}
+	{{if .Issue}}
+		<meta property="og:title" content="{{.Issue.Title}}">
+		<meta property="og:url" content="{{.Issue.HTMLURL}}">
+		{{if .Issue.Content}}
+			<meta property="og:description" content="{{.Issue.Content}}">
+		{{end}}
+	{{else if or .PageIsDiff .IsViewFile}}
+		<meta property="og:title" content="{{.Title}}">
+		<meta property="og:url" content="{{AppUrl}}{{.Link}}">
+		{{if and .PageIsDiff (IsMultilineCommitMessage .Commit.Message)}}
+			<meta property="og:description" content="{{RenderCommitBody $.Context .Commit.Message $.RepoLink $.Repository.ComposeMetas}}">
+		{{end}}
```

Comment:
I do not think it is right to render the text as HTML and use it as `og:description`:  A one to two sentence description of your object.

Reply:
The issue og metadata also dumps the entire issue description in there

Reply:
1. `issue og metadata` dumps (not ideal, but it doesn't mean it is right and the code should be copied)
2.`issue og metadata` only dumps `Content` , which is plain text, not HTML